apps/web/src/scenes/CommunityPage/CommunityMetadataFormModal.tsx

```diff
@@ -0,0 +1,136 @@
+import { useCallback, useState } from 'react';
+import { graphql, useFragment } from 'react-relay';
+import { contexts } from 'shared/analytics/constants';
+import styled from 'styled-components';
+
+import breakpoints from '~/components/core/breakpoints';
+import { Button } from '~/components/core/Button/Button';
+import { VStack } from '~/components/core/Spacer/Stack';
+import { BaseM } from '~/components/core/Text/Text';
+import { TextArea } from '~/components/core/TextArea/TextArea';
+import { useModalActions } from '~/contexts/modal/ModalContext';
+import { useToastActions } from '~/contexts/toast/ToastContext';
+import { CommunityMetadataFormModalFragment$key } from '~/generated/CommunityMetadataFormModalFragment.graphql';
+import colors from '~/shared/theme/colors';
+
+type Props = {
+  communityRef: CommunityMetadataFormModalFragment$key;
+};
+
+export function CommunityMetadataFormModal({ communityRef }: Props) {
+  const community = useFragment(
+    graphql`
+      fragment CommunityMetadataFormModalFragment on Community {
+        dbid
+        name
+      }
+    `,
+    communityRef
+  );
+
+  const { hideModal } = useModalActions();
+  const { pushToast } = useToastActions();
+
+  const [message, setMessage] = useState<string>('');
+  const [status, setStatus] = useState<'SUBMITTING' | 'SUCCESS' | 'ERROR' | undefined>();
+
+  const handleSubmit = useCallback(async () => {
+    if (!message) return;
+    setStatus('SUBMITTING');
+
+    try {
+      const response = await fetch(
+        `https://formspree.io/f/${process.env.NEXT_PUBLIC_FORMSPEE_REQUEST_COLLECTION_ID}`,
+        {
+          method: 'POST',
+          headers: {
+            'Content-Type': 'application/json',
+          },
+          body: JSON.stringify({
+            communityId: community.dbid,
+            communityName: community.name,
+            message,
```

Comment:
nice!! could you also include the submitter's user dbid? 🙏

Reply:
Added userId!